docs/wiki/DEX-Trading-Account-Management.md

```diff
@@ -98,7 +186,9 @@ To manage your DEX trading account tier, here's what you need to do:
 3. Choose the asset for your fidelity bonds and set your `Target Tier`.
 4. Click `Submit` to update your bond options.
 
-**Note**: Make sure you have enough funds to cover your desired `Target Tier`.
+**Note**: Make sure you have enough funds (bond amount * target tier * 2[bond
+overlap to ensure you always have enough bonds to allow you trade] + fees) to
+cover your desired `Target Tier`.
```

Comment:
This is probably not helpful. We just need to get our UI right and point them there to see their reserves.